### DISCUSSION TITLE
Autofocus Input

### DISCUSSION THREAD

**Discussion by Hystarea** [Budibase/budibase#9652] [Help] [answered]
I am looking for a way to autofocus a text field input when a page is loaded. The purpose of this is to make searching an internal Budibase db quicker.

**Comment by melohagan** [comment, marked as answer]
Hi @Hystarea 

There's no setting for doing this at the moment, but it's a good idea. It may be hard to implement though depending on how much of this is controlled by the web browser.

Feature requested!   https://github.com/Budibase/budibase/issues/9699

**Comment by ecmonsen** [comment]
I would like this feature as well.
